user-api/overlays/test/deployment-trigger.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: deployment-trigger
+  annotations:
+    argocd.argoproj.io/hook: PostSync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  template:
+    spec:
+      containers:
+        - name: chat-notification
+          image: quay.io/openshift/origin-cli:latest
+          command:
+            - "oc"
```

Comment:
is this automatically authenticated as the default user of which namespace?!

Reply:
not entirely sure about this, would like to test it out.